go/vt/schemadiff/diff_test.go

```diff
@@ -52,6 +53,36 @@ func TestDiffTables(t *testing.T) {
 			fromName: "t",
 			toName:   "t",
 		},
+		{
+			name:     "change of columns, boolean type",
+			from:     "create table t(id int primary key)",
+			to:       "create table t(id int primary key, i int, b boolean)",
+			diff:     "alter table t add column i int, add column b tinyint(1)",
+			cdiff:    "ALTER TABLE `t` ADD COLUMN `i` int, ADD COLUMN `b` tinyint(1)",
+			action:   "alter",
+			fromName: "t",
+			toName:   "t",
+		},
```

Comment:
Please add another test comparing two tables, one with `bool`, the other with `tinyint(1)`, that expects an empty diff.